### PR TITLE
Org image link resolver made with tests

### LINKED ISSUE #1099: Uploaded image for organisation cannot be accessed
**Describe the bug**
Uploaded image cannot be accessed from the frontend for createOrganisation, the image is getting saved in the Databse and File System though. This is because of the relative url sent to the frontend from the backend (Because the files are being served on a different port)

**To Reproduce**

1. Create an organization.
2. Organizations query in `src/resolvers/Query/organizations.ts` returns a relative path.
3. OrganizationsConnection query in `src/resolvers/Query/organizationsConnection.ts` returns a relative path to image.
4. User query in `src/resolvers/Query/organizationsConnection.ts` returns a relative path of the image.

**Expected behavior**
The image url should be absolute url, ex: `http://localhost:4000/images/fileName.png`

**Actual behavior**
The image url is coming to be relative url, ex: `/images/fileName.png`, coz its directly coming from the db.

**Screenshots**

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by rishav-jha-mech**
**What kind of change does this PR introduce?**

BugFix

**Issue Number:**

Fixes #1099 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

Not required

**If relevant, did you update the documentation?**

Not required

**Summary**

This PR, the organization relative image urls are converted to absolute image urls, with the help of image resolver function.

**Does this PR introduce a breaking change?**

No breaking change

**Other information**

Should i update the doc and add info to add the base URL for the media files, because for now there's this constant BASE_URL which has the current url at which the server is running, in constants folder, but i think it should be taken from .env file and can be configured later as per the image storage plan ?

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by palisadoes** [comment]
- Please fix the failing tests. 
- Code test coverage has gone down and the files you have modified have coverage below there minimum threshold.

**Comment by aviraldevv** [comment]
Hey @rishav-jha-mech no need to use `IN_PRODUCTION!== true`, instead just mock `requestContext.translate()` in the tests. It will fix the failing tests.

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by rishav-jha-mech** [comment]
This should not have happened, what husky is doing 😭

File: src/constants.ts [hunk 1]
```diff
@@ -193,6 +193,8 @@ export const EMAIL_ALREADY_EXISTS_CODE = "email.alreadyExists";
 export const EMAIL_ALREADY_EXISTS_MESSAGE = "email.alreadyExists";
 export const EMAIL_ALREADY_EXISTS_PARAM = "email";
 
+export const BASE_URL = "http://localhost:4000/";
```
Comment (xoldyckk) [inline src/constants.ts:196]: remove this

File: src/resolvers/Organization/image.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { BASE_URL } from "../../constants";
+import { Organization } from "../../models/Organization";
+import { OrganizationResolvers } from "../../types/generatedGraphQLTypes";
+
+export const image: OrganizationResolvers["image"] = async (parent) => {
+  const org = await Organization.findOne({
+    _id: parent._id,
+  });
+  if (org!.image) {
+    return `${BASE_URL}${org!.image}`;
```
Comment (xoldyckk) [inline src/resolvers/Organization/image.ts:10]: Hardcode this to `http://localhost:${process.env.PORT || 4000}/${org!.image}`.

Or if you want you can export `PORT` as a constant from `constants.ts` and use it here. Also, remember that this is just a temporary solution.

File: tests/resolvers/Organization/images.spec.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import "dotenv/config";
```
Comment (xoldyckk) [inline tests/resolvers/Organization/images.spec.ts:1]: Change name of file to `image.spec.ts`

File: src/resolvers/Organization/image.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { BASE_URL } from "../../constants";
+import { Organization } from "../../models/Organization";
+import { OrganizationResolvers } from "../../types/generatedGraphQLTypes";
+
+export const image: OrganizationResolvers["image"] = async (parent) => {
+  const org = await Organization.findOne({
```
Comment (xoldyckk) [inline src/resolvers/Organization/image.ts:6]: You don't need to fetch organization object again. It's already fetched in the parent resolver. Make use of `parent.image` to get the image.

File: src/resolvers/Organization/image.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { BASE_URL } from "../../constants";
+import { OrganizationResolvers } from "../../types/generatedGraphQLTypes";
+
```
Comment (xoldyckk) [inline src/resolvers/Organization/image.ts:3]: Add a comment here explaining why this was needed so people looking at this won't be confused.

File: src/constants.ts (hunk 1 above)
Comment (xoldyckk) [inline src/constants.ts:196]: You can store the `BASE_URL` like this, but if someone starts the server with a `process.env.PORT` other than `4000`, this won't work.

**Review by xoldyckk — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**